libkbfs/kbfs_ops_test.go

```diff
@@ -4796,6 +4800,10 @@ func TestSyncDirtyMultiBlocksSplitInBlockSuccess(t *testing.T) {
 	config.mockDirtyBcache.EXPECT().Get(gomock.Any(),
 		ptrMatcher{fileNode.BlockPointer}, p.Branch).
 		AnyTimes().Return(fileBlock, nil)
+	config.mockBcache.EXPECT().Get(ptrMatcher{node.BlockPointer}).
+		Times(1).Return(rootBlock, nil)
```

Comment:
`Times(1)` is unnecessary here and below, but if you prefer it for some reason, I'm ok with it.